upup/models/cloudup/resources/addons/networking.calico/v1.6.20161206.yaml

```diff
@@ -0,0 +1,307 @@
+# This ConfigMap is used to configure a self-hosted Calico installation.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: calico-config
+  namespace: kube-system
+data:
+  # The calico-etcd PetSet service IP:port
+  etcd_endpoints: "http://100.64.0.136:6666"
+
+  # True enables BGP networking, false tells Calico to enforce
+  # policy only, using native networking.
+  enable_bgp: "true"
+
+  # The CNI network configuration to install on each node.
+  cni_network_config: |-
+    {
+        "name": "k8s-pod-network",
+        "type": "calico",
+        "etcd_endpoints": "__ETCD_ENDPOINTS__",
+        "log_level": "info",
+        "ipam": {
+            "type": "calico-ipam"
+        },
+        "policy": {
+            "type": "k8s",
+             "k8s_api_root": "https://__KUBERNETES_SERVICE_HOST__:__KUBERNETES_SERVICE_PORT__",
+             "k8s_auth_token": "__SERVICEACCOUNT_TOKEN__"
+        },
+        "kubernetes": {
+            "kubeconfig": "/etc/cni/net.d/__KUBECONFIG_FILENAME__"
+        }
+    }
+
+---
+
+# This PetSet lets us run a self-hosted Calico install and not interfere with the
+# cluster etcd, and not need to be aware of where it is when the cluster is coming up
+apiVersion: apps/v1alpha1
```

Comment:
So this is in alpha 1.3.x and is not available in many clusters.  We need to document this, and also update this when 1.5 is out.